docs/installation.rst

```diff
@@ -51,7 +51,8 @@ Installing from npm
 
 You can install CasperJS using `npm <http://npmjs.org/>`_::
 
-    $ npm install -g casperjs
```

Comment:
@istr is there something to do to make this command choose 1.1.0-beta4 ? I dont know how node package manager work.